game-core/src/main/java/games/strategy/triplea/ui/history/HistoryDetailsPanel.java

```diff
@@ -35,32 +38,39 @@
  * </ul>
  */
 public class HistoryDetailsPanel extends JPanel {
-  private static final long serialVersionUID = 5092004144144006960L;
```

Comment:
nit: changing serialVersionUID strings of swing components should not be necessary. I'm not sure there is consensus whether we should default `serialVersoinId` to a simple `1L`. In essence, I'd recommend to not go out of your way to change them unless it's intentional/necessary.

Reply:
Reverted. I wonder if there's some project setting we could set to disable warnings about these missing members for swing classes? The only reason we have these useless things is to silence the warnings AFAIK.